x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/action/user/GetUserPrivilegesResponseTests.java

```diff
@@ -121,31 +189,16 @@ private <T> Set<T> maybeMutate(int random, int index, Set<T> original, Supplier<
     }
 
     private GetUserPrivilegesResponse randomResponse() {
+        return randomResponse(false);
+    }
+
+    private GetUserPrivilegesResponse randomResponse(boolean allowRemoteIndices) {
```

Comment:
Nit: Rename to randomResponseRcs2? I found overloading the method to be confusing while reading the code. It would also be unclear when writing new code using auto-complete, but appending the feature name to the method could make it clearer in PR reviews and future code.

Reply:
Happy to add a `randomResponseWithRemoteIndices()` but would prefer not to introduce Rcs2 in the code. It's more of a project name than a code-level concept.

Reply:
Replying on this [below](https://github.com/elastic/elasticsearch/pull/90826#discussion_r1014227787)